sample-app/src/main/java/com/joinforage/android/example/ui/pos/data/POSUIState.kt

```diff
@@ -4,6 +4,7 @@ import com.joinforage.android.example.network.model.tokenize.PaymentMethod
 import com.joinforage.android.example.ui.pos.MerchantDetailsState
 
 data class POSUIState(
+    val terminalId: String? = "tempDevTerminalId",
```

Comment:
this is great! love that the default value lives here too!

Reply:
do you care if it's `terminalId: String` (non-nullable) or is nullable preferred here?

Reply:
bringing this over from slack for posterity:
> Originally I wanted a way to differentiate between “we’re using the fallback value” and “something went wrong grabbing the value from CPay” because on an actual terminal if we didn’t grab the ID it might be a problem. But also maybe it doesn’t matter and we can just have it always be a string haha